lib/hash_tests.js

```diff
@@ -0,0 +1,33 @@
+var expect = require('chai').expect;
+var fh = require('fh-mbaas-api');
+
+/**
+ * Creates a hash according to a given algorithm and checks its type and length
+ * @param algorithm {string}  MD5/SHA1/SHA256/SHA512
+ * @param length    {number}  Length of the hash that should be created
+ * @param done      {object}
+ */
+function performHashCheck(algorithm, length, done) {
+  fh.hash({
+    'algorithm': algorithm,
+    'text': 'Text to be hashed.'
+  }, function(err, res) {
+    try {
+      expect(err).to.not.exist;
+      expect(res.hashvalue)
+        .to.be.a('string')
+        .to.have.lengthOf(length);
```

Comment:
Should we do an equals check to verify the hashed value is as expected? e.g. use a fixed string and compare the output of fh.hash to what we expect the hashed value to be?